src/gibbs.c

```diff
@@ -666,15 +669,22 @@ SEXP collapsedGibbsSampler(SEXP documents,
 
     for (ii = 0; ii < length(initial_); ++ii) {
       if (!strcmp(CHAR(STRING_ELT(names, ii)), "assignments")) {
-        initial = VECTOR_ELT(initial_, ii);
-        CHECKLEN(initial, NewList, nd);
+    	  // srecko commented this
+    	  //initial = VECTOR_ELT(initial_, ii);
```

Comment:
I believe this can be restored to its original state now that you're using net.assignments.{left,right} for the assignment field right?

Reply:
that is also true